scripts/validate_container.sh

```diff
@@ -81,7 +81,7 @@ available_backends=`\
         qpu=${platform##* }
         requirements=$(cat $file | grep "gpu-requirements:")
         gpus=${requirements##* }
-        if [ "${qpu}" != "remote_rest" ] && [ "${qpu}" != "orca" ] && [ "${qpu}" != "NvcfSimulatorQPU" ] \
+        if [ "${qpu}" != "remote_rest" ] && [ "${qpu}" != "fermioniq" ] && [ "${qpu}" != "orca" ] && [ "${qpu}" != "NvcfSimulatorQPU" ] \
```

Comment:
Since we don't run these examples in the CI, could you please add a job to the `.github/workflows/integration_tests.yml` to run those? 

This is a [sample](https://github.com/NVIDIA/cuda-quantum/blob/b0396b6b75cf3dfcf72515b6d0c249a8085f44c0/.github/workflows/integration_tests.yml#L486) for reference.

Reply:
I added this.